### PR TITLE
Merge JBoss EAP roles from Rho

### LINKED ISSUE #356: Merge jboss_eap fingerprints from rho to sonar
Part of #307 .

### PR DESCRIPTION
**Pull request by noahl**
Also add infrastructure to allow postprocessing of results, which the
JBoss EAP scans take advantage of.

Closes #356 .

### CODE REVIEW

**Comment by ilyavw** [comment]
This looks good to me, the code seems fairly straightforward for the post processing, and I didn't have any problem following the logic of it. That said, I'll leave it with someone a bit more familiar with the code base to give it the ✅

**Comment by noahl** [comment]
I just pushed a commit with more test cases. I think it will push coveralls into the green (as well as make the code more stable!).

File: quipucords/scanner/network/inspect_callback.py [hunk 1]
```diff
@@ -92,6 +93,8 @@ def _finalize_host(self, host):
         logger.debug('host scan complete for %s with facts %s',
                      host, facts)
 
+        results = initial_processing.process(facts)
```
Comment (chambridge) [inline quipucords/scanner/network/inspect_callback.py]: think we should move the above logging statement after this one a print results.

File: quipucords/scanner/network/initial_processing.py [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+        raise NotImplementedError()
+
+
+# #### Processors ####
+
+FIND_WARNING = 'find: WARNING: Hard link count is wrong for /proc: this may' \
+               ' be a bug in your filesystem driver.'
+
+
+class ProcessJbossEapRunningPaths(Processor):
```
Comment (chambridge) [inline quipucords/scanner/network/initial_processing.py]: Maybe break these out into a `scanner/network/eap_processing/__init__.py`? and keep the common ones in here?

Or move this file to `scanner/network/processing/__init__.py` and create and `eap.py` with the eap specific ones?

**Pull request merged**